### ISSUE TITLE
Import includes duplicate groups

### ISSUE THREAD

**Issue by jknight001** [Morikko/sync-tab-groups#92]
If a user deletes a group accidentally and wants to re-import it from the backup, the import will include all groups, even those that are identical to the current ones.  I think it would be better if when doing an import, groups that are identical (same name, same tabs) would not be imported so that the user does not end up with multiple copies of the same group.

For example if one accidentally deletes GroupB:
Before backup import looks like:
GroupA(10)
GroupC(5)

After backup import looks like:
GroupA(10)
GroupA(10)
GroupB(3)
GroupC(5)
GroupC(5)

The extra GroupA and GroupC are unnecessary. Only looking to restore GroupB.

**Comment by Morikko** [comment]
Yes, this can lead to many groups. I won't filter if identical, but I will add a menu to select the groups/tabs you want to import/export.

**Comment by Morikko** [comment]
In v0.6.3, you can select which groups/tabs you want to import/export. However, they are still added as a new groups.
